d2/src/main/java/com/linkedin/d2/xds/XdsClientImpl.java

```diff
@@ -1038,40 +1088,75 @@ private boolean shouldSubscribeUriGlobCollection(ResourceType type) {
     return _subscribeToUriGlobCollection && type == ResourceType.D2_URI_MAP;
   }
 
-  final class RpcRetryTask implements Runnable
-  {
+  final class RpcRetryTask implements Runnable {
     @Override
-    public void run()
-    {
+    public void run() {
       startRpcStreamLocal();
-      for (ResourceType originalType : getResourceSubscribers().keySet())
-      {
-        Set<String> resources = new HashSet<>(getResourceSubscriberMap(originalType).keySet());
-        if (resources.isEmpty())
-        {
+      for (ResourceType originalType : getResourceSubscribers().keySet()) {
+        Map<String, ResourceSubscriber> subscribers = getResourceSubscriberMap(originalType);
+        Map<String, String> irv;
+        Set<String> resources = new HashSet<>(subscribers.keySet());
+        if (resources.isEmpty()) {
           continue;
         }
```

Comment:
Make sure to respect the local formatting:
```suggestion
      for (ResourceType originalType : getResourceSubscribers().keySet())
      {
        Map<String, ResourceSubscriber> subscribers = getResourceSubscriberMap(originalType);
        Map<String, String> irv;
        Set<String> resources = new HashSet<>(subscribers.keySet());
        if (resources.isEmpty())
        {
          continue;
        }
```